### PR TITLE
KFSPTS-7606 removing budget construction references from check reconciliation report

### PR DESCRIPTION
**Pull request by jhulslander**

### CODE REVIEW

File: src/main/java/com/rsmart/kuali/kfs/cr/CRConstants.java [hunk 1]
```diff
@@ -104,6 +104,5 @@
     public static final String SRC_NOT_FOUND = "CR_SOURCE_FOR_NOT_FOUND";
     public static final String BNK_CD_NOT_FOUND = "CR_NOT_FOUND_BANK_CD";
 
-    public static final String BC_REPORT_TEMPLATE_CLASSPATH = "org/kuali/kfs/module/bc/report/";
-    public static final String BC_REPORT_MESSAGES_CLASSPATH = BC_REPORT_TEMPLATE_CLASSPATH + "BudgetOrgReport";
+    public static final String CHECK_RECONCILIATION_REPORT_MESSAGES_CLASSPATH = "com/rsmart/kuali/kfs/cr/report/CRReport";
```
Comment (bhutchinson) [inline src/main/java/com/rsmart/kuali/kfs/cr/CRConstants.java]: Did you need this new one, or could you just use the existing REPORT_MESSAGES_CLASSPATH constant?

**Comment by jhulslander** [comment]
I was able to use the existing variable as mentioned.  That variable wasn't referenced anywhere else, so I felt comfortable changing it to meet the needs of recon report

**Comment by jhulslander** [comment]
this is ready for review now

**Comment by jhulslander** [comment]
retest this please

**Comment by jhulslander** [comment]
Thanks

**Pull request merged**